src/gui/search/searchwidget.cpp

```diff
@@ -141,6 +144,21 @@ SearchWidget::SearchWidget(MainWindow *mainWindow)
     connect(focusSearchHotkey, &QShortcut::activated, this, &SearchWidget::toggleFocusBetweenLineEdits);
 }
 
+bool SearchWidget::eventFilter(QObject *object, QEvent *event)
+{
+    Q_UNUSED(object);
```

Comment:
You still should check whether object is `m_ui->tabWidget->tabBar()`.

Reply:
I would use reference example from https://doc.qt.io/qt-5/qobject.html#eventFilter as a basis.